Makefile

```diff
@@ -534,11 +534,9 @@ validate-runtime-image: bin/kubectl
 		fi; \
 		if [ $$cmd == "python3" ]; then \
 			echo "=> Checking notebook execution..." ; \
-			$(KUBECTL_BIN) exec runtime-pod -- /bin/sh -c "python3 -m pip install -r /opt/app-root/elyra/requirements-elyra.txt && \
-				curl https://raw.githubusercontent.com/nteract/papermill/main/papermill/tests/notebooks/simple_execute.ipynb --output simple_execute.ipynb && \
-				python3 -m papermill simple_execute.ipynb output.ipynb > /dev/null" ; \
+			$(KUBECTL_BIN) exec runtime-pod -- /bin/sh -c "python3 -m pip install pipenv --three sync && \
```

Comment:
```
$(KUBECTL_BIN) exec runtime-pod -- /bin/sh -c "curl https://raw.githubusercontent.com/opendatahub-io/elyra/refs/heads/main/etc/generic/requirements-elyra.txt --output req.txt && \ python3 -m pip install -r req.txt && \
				curl https://raw.githubusercontent.com/nteract/papermill/main/papermill/tests/notebooks/simple_execute.ipynb --output simple_execute.ipynb && \
				python3 -m papermill simple_execute.ipynb output.ipynb > /dev/null" ; \
```

Try this.
As, the reason for using the requirements.txt is so that , we can make sure that packages are installable , 
perhaps, we can extend this in further tasks, to do some extra checks.